### PR TITLE
ENG-735: update collab section

### PR DESCRIPTION
**Pull request by Richard-Burd**
The 'Collab' section is transformed into a 'Build' section as we no longer want to display our collab partners on the homepage.

### CODE REVIEW

File: web/src/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -159,47 +121,26 @@ const HomePage = () => {
...
-                    alt={item.alt}
-                    width={150}
-                    height={150}
-                  />
-                </a>
-              ))}
-            </div>
+      {/* Build Section: we want a link to this in Header.tsx */}
+      <Section className="my-20" id="build">
+        <div className="mx-auto max-w-xl">
```
Comment (stormcloud266) [inline web/src/pages/HomePage/HomePage.tsx]: Let's make this section have the same max width as the section above

File: web/src/pages/HomePage/HomePage.tsx [hunk 2]
```diff
@@ -159,47 +121,26 @@ const HomePage = () => {
...
+          <Heading>Build Your Own Gamified Treasure Hunts</Heading>
+          <Text>
+            Use Infinity Keys to create incentivized games for any community.
+            Build experiences with the IK no-code platform:
+          </Text>
+            <ul className="list-disc ml-6 space-y-4 md:space-y-0 text-indigo-200 ">
```
Comment (stormcloud266) [inline web/src/pages/HomePage/HomePage.tsx]: Can you add the same margin top to this list as well

File: web/src/pages/HomePage/HomePage.tsx [hunk 3]
```diff
@@ -159,47 +121,26 @@ const HomePage = () => {
...
+              <li>Embeddable videos, images, or games</li>
+              <li>Secret passcodes</li>
+              <li>Gated access using digital collectibles</li>
+            </ul>
+          <div className="flicker-container mt-4 text-2xl">
+            <Link to="/packs">
+              <Flicker delay=".6s">See Demo Games</Flicker>
+            </Link>
```
Comment (stormcloud266) [inline web/src/pages/HomePage/HomePage.tsx]: Checkout how Redwood does [named routes](https://redwoodjs.com/docs/router#link-and-named-route-functions). It makes it so we can change path in `Routes` and not have to change any of our `Link` hrefs.

This looks rad btw

**Review by stormcloud266 — CHANGES_REQUESTED** [review]

File: web/src/components/Header/Header.tsx [hunk 4]
```diff
@@ -59,32 +49,46 @@ const Header = () => {
...
-                      </a>
-                    )
-                  })}
+                  <Link
+                    to={routes.home()}
+                    className="header-nav--link text-2xl font-medium text-white hover:text-turquoise"
+                  >
+                    Home
+                  </Link>
+                  <a
```
Comment (illepic) [inline web/src/components/Header/Header.tsx:58]: Let's add `rel=noopener`, `re for each a tag.

File: web/src/pages/HomePage/HomePage.tsx [hunk 5]
```diff
@@ -16,58 +15,37 @@ import Section from 'src/components/Section/Section'
...
 
+  const buildSection = useRef<HTMLDivElement | null >(null)
+
+  const { hash } = useLocation()
+
+
+  useEffect(() => {
+    if (buildSection?.current && hash === '#build') {
+      setTimeout(() => {
+        console.log("we're at line 42 right here")
```
Comment (illepic) [inline web/src/pages/HomePage/HomePage.tsx]: Nuke the console.log.

File: web/src/pages/HomePage/HomePage.tsx [hunk 6]
```diff
@@ -16,58 +15,37 @@ import Section from 'src/components/Section/Section'
 import Seo from 'src/components/Seo/Seo'
 import Text from 'src/components/Text/Text'
 import Wrapper from 'src/components/Wrapper/Wrapper'
+import { Link, routes, useLocation } from '@redwoodjs/router'
 import { heroStars } from 'src/lib/animations/hero-stars'
 
 import '@infinity-keys/react-lens-share-button/dist/style.css'
+import CollapsibleMarkdownStories from 'src/components/CollapsibleMarkdown/CollapsibleMarkdown.stories'
```
Comment (illepic) [inline web/src/pages/HomePage/HomePage.tsx]: Remove, this was accidental import.

**Review by illepic — CHANGES_REQUESTED** [review]

**Review by illepic — APPROVED** [review]

**Pull request merged**